inbm/dispatcher-agent/tests/unit/source/test_source_command.py

```diff
@@ -164,9 +164,9 @@ def test_do_source_command_remove(
             "dispatcher.source.source_command.create_application_source_manager",
             OsType.Ubuntu,
             ApplicationAddSourceParameters(
```

Comment:
Ensure that the test cases are updated to reflect the new parameter order in `ApplicationAddSourceParameters` and `ApplicationUpdateSourceParameters`. [important]